data/soapvalidator/MailClient/HAB/ZCS-5395_DeleteHABGroups.xml

```diff
@@ -0,0 +1,788 @@
+		</t:test>
+</t:test_case>
+
+	<t:test_case testcaseid="DeleteHABGroups_02" type="smoke"
+		bugids="ZCS-5395">
+		<t:objective>DeleteHABgroups for a dynamic group with
+			cascadeDelete="true"
+		</t:objective>
+		<t:steps>
+			1. Fire DeleteHABGorups request for static group4 and
+			cascadeDelete="true"
+			2. Verify request should succeed and group7 which
+			is dynamic subgroup should also get deleted
+		</t:steps>
+
+		<t:test>
+			<t:request>
+				<DeleteDistributionListRequest id="${hab.group4_id}"
+					cascadeDelete="true" xmlns="urn:zimbraAdmin" />
+			</t:request>
+			<t:response>
+				<t:select path="//admin:DeleteDistributionListResponse" />
+			</t:response>
+		</t:test>
+
+		<t:test>
+			<t:request>
+				<GetDistributionListRequest xmlns="urn:zimbraAdmin">
+					<dl by="id">${hab.group7_id}</dl>
+				</GetDistributionListRequest>
+			</t:request>
+			<t:response>
+				<t:select path="//zimbra:Code" match="^account.NO_SUCH_DISTRIBUTION_LIST" />
+			</t:response>
+		</t:test>
+</t:test_case>
+
+	<t:test_case testcaseid="DeleteHABGroups_03" type="smoke"
+		bugids="ZCS-5395">
+		<t:objective>DeleteHABgroups for a dynamic group with
+			cascadeDelete="false"
+		</t:objective>
+		<t:steps>
+			1. create a new dynamic group7.
+			1. Fire DeleteHABGorups request for
+			dynamic group7 and cascadeDelete="false"
+			2. Verify request should
+			succeed and group7 which is dynamic group should get deleted
+		</t:steps>
+
+		<t:test>
+			<t:request>
+				<CreateHABGroupRequest habDisplayName="${hab.group7_name}"
+					habOrgUnit="${ou1.name}" name="${hab.group7}" dynamic="1"
+					xmlns="urn:zimbraAdmin">
+					<a n="zimbraNotes">${hab.notes}</a>
+					<a n="memberURL">${member_url}</a>
+				</CreateHABGroupRequest>
+			</t:request>
+			<t:response>
+				<t:select path="//admin:CreateHABGroupResponse//admin:dl"
+					attr="name" match="${hab.group7}" />
+				<t:select path="//admin:CreateHABGroupResponse//admin:dl"
+					attr="id" set="hab.group7_id" />
+				<t:select
+					path="//admin:CreateHABGroupResponse//admin:a[@n='displayName']"
+					match="${hab.group7_name}" />
+			</t:response>
+		</t:test>
+
+		<t:test>
+			<t:request>
+				<DeleteDistributionListRequest id="${hab.group7_id}"
+					cascadeDelete="false" xmlns="urn:zimbraAdmin" />
+			</t:request>
+			<t:response>
+				<t:select path="//admin:DeleteDistributionListResponse" />
+			</t:response>
+		</t:test>
+
+		<t:test>
+			<t:request>
+				<GetDistributionListRequest xmlns="urn:zimbraAdmin">
+					<dl by="id">${hab.group7_id}</dl>
+				</GetDistributionListRequest>
+			</t:request>
+			<t:response>
+				<t:select path="//zimbra:Code" match="^account.NO_SUCH_DISTRIBUTION_LIST" />
+			</t:response>
+		</t:test>
+</t:test_case>
+
+	<t:test_case testcaseid="DeleteHABGroups_04" type="smoke"
+		bugids="ZCS-5395">
+		<t:objective>DeleteHABgroups for a dynamic sub group with
+			cascadeDelete="true"
+		</t:objective>
+		<t:steps>
+			1. Add a dynamic subgroup to a static group.
```

Comment:
where is the req to add dynamic group to static

Reply:
added